GliaWidgets/Color.swift

```diff
@@ -8,7 +8,7 @@ enum Color {
     static let baseNormal = UIColor(hex: 0x6C7683) // grey
     static let baseShade = UIColor(hex: 0x6C7683, alpha: 0.5)
     static let baseDark = UIColor(hex: 0x2C0735) // purple
-    static let lightGrey = UIColor(hex: 0xF3F3F3) // light gray
+    static let baseNeutral = UIColor(hex: 0xF3F3F3) // light gray
     static let grey = UIColor(hex: 0x999999) // gray
     static let background: UIColor = .white
```

Comment:
Let's try to remove `grey`, `background` colors and align `GlobalColors.baseLight` with `Color.baseLight` and `GlobalColors.background` with `Color.neutral`

Reply:
fixed [f8b8b60](https://github.com/salemove/ios-sdk-widgets/pull/777/commits/f8b8b600ed2c3cccef497d0c35a59dee80a4cfea)